sdk/confidentialledger/arm-confidentialledger/src/models/index.ts

```diff
@@ -224,12 +224,91 @@ export interface ConfidentialLedgerList {
   nextLink?: string;
 }
 
+/** Additional Managed CCF properties. */
+export interface ManagedCCFProperties {
+  /**
+   * Unique name for the Managed CCF.
+   * NOTE: This property will not be serialized. It can only be populated by the server.
+   */
+  readonly appName?: string;
+  /**
+   * Endpoint for calling Managed CCF Service.
+   * NOTE: This property will not be serialized. It can only be populated by the server.
+   */
+  readonly appUri?: string;
+  /**
+   * Endpoint for accessing network identity.
+   * NOTE: This property will not be serialized. It can only be populated by the server.
+   */
+  readonly identityServiceUri?: string;
+  /** List of member identity certificates for  Managed CCF */
+  memberIdentityCertificates?: MemberIdentityCertificate[];
+  /** Deployment Type of Managed CCF */
+  deploymentType?: DeploymentType;
+  /**
+   * Provisioning state of Ledger Resource
+   * NOTE: This property will not be serialized. It can only be populated by the server.
+   */
+  readonly provisioningState?: ProvisioningState;
+}
+
+/** Object representing MemberIdentityCertificate for Managed CCF. */
+export interface MemberIdentityCertificate {
+  /** Member Identity Certificate */
+  certificate?: string;
+  /** Member Identity Certificate Encryption Key */
+  encryptionkey?: string;
+  /** Anything */
+  tags?: any;
+}
+
+/** Object representing DeploymentType for Managed CCF. */
+export interface DeploymentType {
+  /** Unique name for the Managed CCF. */
+  languageRuntime?: LanguageRuntime;
+  /** Source Uri containing ManagedCCF code */
+  appSourceUri?: string;
+}
+
+/** Object that includes an array of Managed CCF and a possible link for next set. */
+export interface ManagedCCFList {
+  /** List of Managed CCF */
+  value?: ManagedCCF[];
+  /** The URL the client should use to fetch the next page (per server side paging). */
+  nextLink?: string;
+}
+
+/** Tags for Managed CCF Certificates */
+export interface CertificateTags {
+  /** Additional tags for Managed CCF Certificates */
+  tags?: { [propertyName: string]: string };
+}
+
+/** The resource model definition for an Azure Resource Manager tracked top level resource which has 'tags' and a 'location' */
+export interface TrackedResource extends Resource {
+  /** Resource tags. */
+  tags?: { [propertyName: string]: string };
+  /** The geo-location where the resource lives */
+  location: string;
+}
+
 /** Confidential Ledger. Contains the properties of Confidential Ledger Resource. */
-export interface ConfidentialLedger extends Resource, ResourceLocation, Tags {
+export interface ConfidentialLedger
+  extends TrackedResource,
+    ResourceLocation,
```

Comment:
But in `ResourceLocation` the property location is optional:
https://github.com/azure-sdk/azure-sdk-for-js/blob/7af0640babd5b3bb99021d4b3683214ab02acbf1/sdk/confidentialledger/arm-confidentialledger/src/models/index.ts#L208